myset.h

```diff
@@ -0,0 +1,156 @@
+//
+// Created by Kinstintin on 28.04.2024.
+//
+
+#ifndef UNTITLED_MYSET_H
+#define UNTITLED_MYSET_H
+#include <random>
+#include <unordered_map>
+
+static std::uniform_int_distribution<int> uniform(1, INT32_MAX);
+std::random_device dev;
+static std::mt19937_64 rng(dev());
+
+
+template <typename T,
+        typename Compare = std::less<T>>
+class Myset{
+private:
+    struct Node{
+        Node* par;
+        Node* minim;
+        Node* left;
+        Node* right;
+        T key;
+        int prior;
+        int count;
+        explicit Node(const T &value){
```

Comment:
Use list initialization

Reply:
Could you please specify why exactly I should do it?